singer-io/tap-looker
# v.0.0.4 Error Handling

# Description of change
Minor changes for error handling when `sync.py` GET requests child endpoints, and the associated data for the parent_id is either missing or not-accessible errors. These result in 404 and 5xx Errors. The tap will re-try and log these warnings, but it will not fail now.

# Manual QA steps
Ran singer-discover, singer-check-tap, and target-stitch for all endpoints. No issues.
 
# Risks
Low. Minor change. Currently in Alpha testing for Bytecode.
 
# Rollback steps
Revert to v.0.0.3
